packages/repo-tools/templates/typescript-backstage/api.mustache

```diff
@@ -9,7 +9,12 @@ import * as parser from 'uri-template';
 import { {{classname}} } from '{{filename}}.model{{importFileExtension}}';
 {{/imports}}
 
-type TypedResponse<T> = Omit<Response, 'json'> & {
+/**
+ * Wraps the Response type to convey a type on the json call.
+ *
+ * @oublic
```

Comment:
```suggestion
 * @public
```

Reply:
🤦 

that actually made me find one more such occurrence in docs